Ibraheem-adeyemo/instatutor
# #167533878 create home page

#### What does this PR do?
- Create a Home page  which is accessible to everybody
- have the Get '/'  resources working

#### Description of task to be completed
- Have the home page fully created with detailed information about instatutor

#### How should this be manually tested?
After cloning the repo cd into it and run npm install and npm test
- On the browser navigate to /, a fully developed detailed home page should come up


#### What are the relevant pivotal tracker stories
  

[Finshes #167533878](https://www.pivotaltracker.com/story/show/167533878)
